.travis.yml

```diff
@@ -7,6 +7,9 @@ notifications:
   email: false
 node_js:
   - '8'
+  - '6'
+  - '4'
+before_install: if [[ `npm -v` < 3* ]]; then npm i -g npm@^3; fi
```

Comment:
yes it's up to you. but then I think we should specify `"npm": ">=5"` in `engines` field of `package.json` to reflect Travis config